resources/views/welcome.blade.php

```diff
@@ -32,31 +32,39 @@
         </div>
         <x-interpretation name="{{ __('What is the Accessibility Exchange?', [], 'en') }}" />
         <div class="frame">
-            <x-placeholder />
+            <div class="stack w-full" x-data="vimeoPlayer({
+                url: @if (locale() === 'en') 'https://vimeo.com/789855323'
+                    @elseif (locale() === 'fr')
+                    'https://vimeo.com/789772188'
+                    @elseif (locale() === 'asl')
+                    'https://vimeo.com/788810528/73b8c80ad8'
+                    @elseif (locale() === 'lsq')
+                    'https://vimeo.com/789829171' @endif,
+                byline: false,
+                pip: true,
+                portrait: false,
+                responsive: true,
+                speed: true,
+                title: false
+            })" @ended="player().setCurrentTime(0)">
+            </div>
         </div>
     </section>
 
     <section class="stack stack:lg" aria-labelledby="how">
         <h2 class="text-center" id="how">{{ __('How does this work?') }}</h2>
 
-        <p>{{ __('This site is for three kinds of users. Select an option below to learn more.') }}</p>
+        <p class="text-center">{{ __('This site is for three kinds of users. Select an option below to learn more.') }}
+        </p>
         <x-interpretation name="{{ __('How does this work?', [], 'en') }}" />
-        <x-media-text mediaWidth="md:w-2/3" textWidth="md:w-1/3" first="text">
-            <x-slot name="media">
-                <x-placeholder width="915" height="515" />
-            </x-slot>
+        <div class="grid">
             <div class="flex h-full flex-col justify-center">
```

Comment:
Looks like you want these to align to the top, so `justify-center` class should be removed from them.